docs/add-ons/query.md

```diff
@@ -140,3 +140,44 @@ Multiple instances of the {switch=} tag may be used and ExpressionEngine will in
 ### `{total_results}`
 
 The number of total results of the query.
+
+## Troublshooting
+**Column variables are not parsing as expected**  
+When using the Query Module within other tag pairs such as `exp:channel:entries` which have their own variables, a conflict may arise in parsing the `exp:query` tag. 
+
```

Comment:
I think it's supposed to be 'that' instead of 'which'. And I think your example does a good job of carrying the heavy lifting explaining things, so it might be better to simplify the written explanation.  

> When using the Query inside another tag pair, variable conflicts may occur.

I think what's there is fine, but I also think without the example, no matter what is written, it won't be clear.